.tool-versions

```diff
@@ -1,2 +1 @@
-nodejs 16.10.0
```

Comment:
Why are we changing the `.tool-versions` so it doesn't handle the node version? It's default for asdf to use it for setting the node version